components/locid_transform/data/config.json

```diff
@@ -5,6 +5,7 @@
       "fallback/parents@1",
       "fallback/supplement/co@1",
       "locid_transform/aliases@1",
```

Comment:
compiled data should not carry both structs, remove v1

Reply:
If I remove it, I lose ability to test for aliasesv1 vs aliasesv2. Do you have a suggestion how to workaround it?

Reply:
Yes, compile a blob provider into `tests/data` and use that for tests. I think we do this in this crate already. As this is a backward compatibility test that's not something that will have to be updated again. Datetime does this for example.

Reply:
done.